src/components/calculator/LegalOfficerCaseCost.tsx

```diff
@@ -31,6 +35,8 @@ export const ZERO_COLLECTION_COST_PARAMETERS: CollectionCostParameters = {
 export const DEFAULT_STANDARD_COLLECTION_COST_PARAMETERS: CollectionCostParameters = {
     custom: false,
     protectedValue: 2500000n,
+    protectedCollectionItemValue: 1250000n,
+    protectedTokensRecordValue: 1250000n,
```

Comment:
I would put a zero default value for the recurrent fees. That rule about recurrent fees being the half of value fees does not make any sense.